### PR TITLE
added 'csrf-field' optional attribute.

### PR DESCRIPTION
**Pull request by eroluysal**

### CODE REVIEW

**Comment by dgreif** [comment]
@eroluysal would you mind providing some context as to why you need this change?

**Comment by eroluysal** [comment]
@dgreif I guess "authenticity_token" is a specific for Rails framework. for example; I want to use auto-complete-element with Symfony framework and Symfony framework uses "_token" parameter name.

File: src/index.ts [hunk 1]
```diff
@@ -62,6 +62,10 @@ export default class AutoCheckElement extends HTMLElement {
     return link.href
   }
 
+  get csrfField(): string {
+    return this.getAttribute('csrf-field') || 'authenticity_token'
+  }
+
```
Comment (koddsson) [inline src/index.ts:68]: Can we move this to below the `required` setter at the end of the class? Right now, it's right between the `src` getter and setter, so it makes it hard to scan for specific functions.

Also, adding a setter for the attribute would be great.

**Review by koddsson — APPROVED** [review]
This change looks good to me after the suggested changes from me.

I wasn't sure if adding `csrfField` to the API for the component makes sense, but I think it does in the end.

File: src/index.ts (hunk 1 above)
Comment (eroluysal) [reply src/index.ts]: thanks for review.

File: src/index.ts [hunk 2]
```diff
@@ -86,6 +86,14 @@ export default class AutoCheckElement extends HTMLElement {
       this.removeAttribute('required')
     }
   }
+
+  get csrfField(): string {
+    return this.getAttribute('csrf-field') || 'authenticity_token'
+  }
+
+  set csrfField(value: string) {
+    this.setAttribute('csrf-field', value);
```
Comment (koddsson) [inline src/index.ts:95]:
```suggestion
    this.setAttribute('csrf-field', value)
```

**Pull request merged**